Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean

```diff
@@ -91,9 +98,21 @@ theorem monotoneOn_posLog : MonotoneOn log⁺ (Set.Ici 0) := by
     simp only [this, and_true, ge_iff_le]
     linarith
 
+/-- The function `log⁺` commutes with taking powers. -/
+@[simp] lemma posLog_pow {n : ℕ} {a : ℝ} :
+    log⁺ (a ^ n) = n * log⁺ a := by
```

Comment:
I think `x` might be more conventional for a real number --- though this file uses a mix of different conventions (sometimes `x`, sometimes `a` and `b`, sometimes `r`). In other words, consider my suggestion optional.
```suggestion
@[simp]
lemma posLog_pow {n : ℕ} {x : ℝ} : log⁺ (x ^ n) = n * log⁺ x := by
```

Reply:
Implemented your suggestion. Also, use `x` consistently throughout the file now.